ada/tasks.py

```diff
@@ -0,0 +1,29 @@
+from celery.exceptions import SoftTimeLimitExceeded
+from django.conf import settings
+from requests.exceptions import RequestException
+from temba_client.exceptions import TembaHttpError
+from temba_client.v2 import TembaClient
+
+from ndoh_hub.celery import app
+
+
+rapidpro = None
+if settings.ADA_RAPIDPRO_URL and settings.ADA_RAPIDPRO_TOKEN:
```

Comment:
Does ADA have its own rapidpro org? not on momconnect?

Reply:
No it doesn't. I wasn't sure this was so I thought it was just a project reference thing. I'll remove